src/components/ActionCards/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { actionCardContent } from '../../constants';
+import './actionCards.css';
+
+const renderIcon = Icon => (
+  <Icon size={50} color="grey" style={{ paddingRight: '10px' }} />
```

Comment:
Do you think we should give this icon a class so that we can move the css from being inlined?